pom.xml

```
@@ -6,7 +6,7 @@
 
 	<groupId>name.valery1707.kaitai</groupId>
 	<artifactId>kaitai-maven-plugin</artifactId>
-	<version>0.1.7-SNAPSHOT</version>
+	<version>0.1.7</version>
```

Please revert this change - the version will be changed on the release process
```suggestion
	<version>0.1.7-SNAPSHOT</version>
```